### ISSUE TITLE
Refactor `user` to another name

### ISSUE THREAD

**Issue by superkhau** [strongloop/loopback-example-access-control#73]
We get a lot of confusion from my original implementation of `user` extending builtin `User`. We should rename it to something like `CustomUser` or something to signify it's obviously not the same as builtin `User`.

**Comment by timotheemoulin** [comment]
Thanks for reporting that ;)

The official documentation tells us to use ``Customer`` or ``Client`` when overriding the ``User`` model.
Moreover, I am not conviced that mixing naming conventions in the example is a good practice.

https://docs.strongloop.com/display/public/LB/Managing+users

**Comment by superkhau** [comment]
Do you have any suggestions? :ear:

**Comment by timotheemoulin** [comment]
I think that using the same names as in the official documentation ``Customer``, ``Client`` would be a good idea. I might highy help people understanding the examples.

**Comment by superkhau** [comment]
Haha, actually `user` was the official name before and we added `Customer`/`Client because we had to keep explaining it before, but never got around to updating the examples to match. ;)

**Comment by timotheemoulin** [comment]
Ok I get it.

I don't mind using ``User`` twice. That's even what I would have done if
not told otherwise in the doc. What bother me more is that it's once
written in lowercase and the second time with an uppercase *U*

**Comment by superkhau** [comment]
No, it has to be lowercase or it will conflict with the builtin `User` model. That or you can name it something other than "User".

**Comment by DaGaMs** [comment]
the `User` vs `user` issue also breaks down if you use mysql as a backend, since it can't distinguish between tables by case.

**Comment by antarasi** [comment]
`Account` seems to be distinguishable enough and independent from actual user purpose (Client, Customer etc)

**Comment by superkhau** [comment]
Would someone like to submit a PR? I can help with the landing. ;)

**Comment by csakis** [comment]
I addressed this issue in a new PR
